homeassistant/components/nina/binary_sensor.py

```diff
@@ -82,15 +87,15 @@ def extra_state_attributes(self) -> dict[str, Any]:
         ) or not self.is_on:
```

Comment:
`is_on` already checks that the warning index is in range now, so you can simplify this by removing that part of the check:

```
        if not self.is_on:
            return {}
```